src/ui/pages/api/auth/[...nextauth].js

```diff
@@ -14,12 +14,14 @@ export const authOptions = {
       // Persist the OAuth access_token to the token right after signin
       if (account) {
         token.accessToken = account.access_token;
+        token.jwt = account.id_token;
```

Comment:
Did you verify that Google is sending back an `id_token`?

Reply:
I did verify locally. Are you worried about the response from Google not including an `id_token`?

Reply:
Yeah, but if you verified all good